### DISCUSSION TITLE
胶囊栏 点击主页 报错

### DISCUSSION THREAD

**Discussion by kukudeliu** [TaleLin/lin-ui#1321] [求助] [answered]
TypeError: Cannot read property 'mark' of undefined

**Comment by juzi214032** [comment, marked as answer]
给个截图看一下

**Reply by kukudeliu** [reply]
[![yRaYJP.md.png](https://s3.ax1x.com/2021/02/18/yRaYJP.md.png)](https://imgchr.com/i/yRaYJP)

**Reply by kukudeliu** [reply]
index.js 让我格式化了，本想自己查查错，没整明白

**Reply by kukudeliu** [reply]
当前页是tabBar

**Reply by juzi214032** [reply]
目前基础库版本是多少 调低试试

**Reply by kukudeliu** [reply]
> 目前基础库版本是多少 调低试试

擦 调低好用了

**Reply by kukudeliu** [reply]
thank you

**Reply by juzi214032** [reply]
你之前用的是哪个版本会报错，我优化一下这个

**Reply by kukudeliu** [reply]
> 你之前用的是哪个版本会报错，我优化一下这个
最新 2.15.0

**Reply by juzi214032** [reply]
ok 下个版本会解决这个报错

**Reply by juzi214032** [reply]
我这边在 2.15.0 没有复现这个问题，方便的话可以提供个代码片段给我测试一下

**Reply by kukudeliu** [reply]
> 我这边在 2.15.0 没有复现这个问题，方便的话可以提供个代码片段给我测试一下

https://developers.weixin.qq.com/s/CU5z9emp7aoY

**Reply by kukudeliu** [reply]
> 我这边在 2.15.0 没有复现这个问题，方便的话可以提供个代码片段给我测试一下

https://developers.weixin.qq.com/s/CU5z9emp7aoY
